src/GitHub.VisualStudio/Styles/GitHubComboBox.xaml

```diff
@@ -0,0 +1,56 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:ui="clr-namespace:GitHub.UI;assembly=GitHub.UI">
+    
+    <Style x:Key="GitHubComboBoxBorder" TargetType="{x:Type Border}">
+        <Setter Property="Margin" Value="10" />
+        <Setter Property="BorderThickness" Value="1" />
+        <Setter Property="BorderBrush" Value="{DynamicResource GitHubComboBoxBorderBrush}" />
+        <Setter Property="Effect">
+            <Setter.Value>
+                <DropShadowEffect Direction="315" ShadowDepth="5" BlurRadius="5" Opacity="0.25" />
+            </Setter.Value>
+        </Setter>
+    </Style>
+
+    <Style x:Key="GitHubComboBoxContainer" TargetType="{x:Type Grid}">
+        <Setter Property="Background" Value="{DynamicResource GitHubComboBoxBackgroundBrush}" />
+
+        <Style.Resources>
+            <Style TargetType="{x:Type TextBlock}" BasedOn="{StaticResource {x:Type TextBlock}}">
+                <Setter Property="Foreground" Value="{DynamicResource GitHubVsToolWindowText}" />
+            </Style>
+
+            <Style TargetType="{x:Type ui:OcticonImage}" BasedOn="{StaticResource {x:Type ui:OcticonImage}}">
```

Comment:
This line triggers a XAML parse error:

> {"Cannot find resource named 'GitHub.UI.OcticonImage'. Resource names are case sensitive."} 

I'm assuming I'm not resourcing correctly in Visual Studio on [Line 3](https://github.com/github/VisualStudio/blob/add-combobox-ui/src/GitHub.VisualStudio/Styles/GitHubComboBox.xaml#L3) :stuck_out_tongue: 

```xaml
<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
                    xmlns:ui="clr-namespace:GitHub.UI;assembly=GitHub.UI">
```

Any chance of pointing me in the right direction of understanding how to get the right resource? I've been trying to follow how we're doing it in `PullRequestCreationView.xaml`.